.github/test-categories/SECURITY_5

```diff
@@ -1,3 +1,4 @@
 com.ibm.websphere.security_fat.1
 com.ibm.ws.security.auth.data_fat
 com.ibm.ws.security.client_fat
+com.ibm.ws.security.wim.adapter.ldap_fat.krb5
```

Comment:
Can we put this in the same group as our regular LDAP fats? Might be more convenient.

Reply:
Move to SECURITY_4